monad-logger/Control/Monad/Logger.hs

```diff
@@ -579,6 +582,22 @@ runStderrLoggingT = (`runLoggingT` defaultOutput stderr)
 runStdoutLoggingT :: MonadIO m => LoggingT m a -> m a
 runStdoutLoggingT = (`runLoggingT` defaultOutput stdout)
 
+-- | Run a block using a @MonadLogger@ instance which writes tuples to an unbounded channel.
```

Comment:
A slightly expanded documentation to explain how this plays together with the other `Chan` functions would be very useful, it's not immediately obvious from reading the docs/types/code.